proto/neutron/dex/tx.proto

```diff
@@ -115,6 +118,12 @@ message MsgPlaceLimitOrder {
     (gogoproto.nullable) = true,
     (gogoproto.jsontag) = "max_amount_out"
   ];
+  string price_in_to_out = 11 [
```

Comment:
Also please:
1. add `price_in_to_out` field to `MsgPlaceLimitOrder` in our [wasm bindings](https://github.com/neutron-org/neutron/blob/6364e0025538acc1efc82d7197dc2a2da373fa1b/wasmbinding/bindings/msg.go#L263).
2. reflect field addition in Neutron CW SDK [itself](https://github.com/neutron-org/neutron-sdk/blob/4fc7cfd63b685753d62e64125bc30e7512a911b1/packages/neutron-sdk/src/bindings/dex/msg.rs#L51).
3. and you will need to regenerate protobufs there to make stargate msgs up to date like i did it [here](https://github.com/neutron-org/neutron-sdk/pull/139).
4. And reflect changes in neutron-dev-contracts (like it was done [here](https://github.com/neutron-org/neutron-dev-contracts/pull/45))
5. And restart integration tests to see that everything is ok with bindings and stargate